examples/lighting-app/silabs/data_model/lighting-thread-app.zap

```diff
@@ -3140,6 +3140,2280 @@
     },
     {
       "id": 2,
+      "name": "Anonymous Endpoint Type",
```

Comment:
Something seems wrong with this zap, we are adding thousands of lines, amongst which mandatory stuff like Access Control or Basic Information Cluster, which are probably already in the first 3000 lines of this file.

So I am not sure about what is going on here. Are we adding a new endpoint?